### ISSUE TITLE
New series type - quiver/cone plots

### ISSUE THREAD

**Issue by julioasotodv** [apache/echarts#14046]
### What problem does this feature solve?
In physics, an essential chart type are quiver (2D) and cone (3D) plots, as they allow for easy representation of vectors (such as force layouts).

Plotly offers these kind of charts both in [2D](https://plotly.com/python/quiver-plots/) and [3D](https://plotly.com/python/cone-plot/), and right now this is the only reason I cannot use echarts.

Adding support for quiver and cone plots would be great.

Thank you!

### What does the proposed API look like?
Basically like any other `series` type.

**Comment by pissang** [comment]
@julioasotodv Hi, for 2D version, you can try to use `custom` series to draw the vectors. I wrote a simple example

https://jsfiddle.net/eL8kujv3/

**Comment by julioasotodv** [comment]
> @julioasotodv Hi, for 2D version, you can try to use `custom` series to draw the vectors. I wrote a simple example
> 
> https://jsfiddle.net/eL8kujv3/

👍

**Comment by julioasotodv** [comment]
> @julioasotodv Hi, for 2D version, you can try to use `custom` series to draw the vectors. I wrote a simple example
> 
> https://jsfiddle.net/eL8kujv3/
👍

**Comment by julioasotodv** [comment]
Thank you @pissang!
